### PR TITLE
Cleanup max

### PR DESCRIPTION
**Pull request by mffrank**

### CODE REVIEW

File: R/importPCPdata.R [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+#' @export
+
+
+importPCPdata <- function(input_data,
+                          fraction_annotation,
+                          rm_decoys = FALSE){
+  
+  ## test arguments
+  if (missing(input_data)){
+    stop("Need to specify input_data in form of a filename or R data.table.")
```
Comment (ibludau) [inline R/importPCPdata.R:37]: Please check if everything still runs and change error messages in test.

**Review by ibludau — CHANGES_REQUESTED** [review]
Please check comments below. I made some changes to the main function - please adjust tests accordingly.

**Comment by mffrank** [comment]
Included recommendations, should be mergable now

**Review by ibludau — APPROVED** [review]
Top :)

**Pull request merged**